test/swe2d/test_thacker.py

```diff
@@ -0,0 +1,60 @@
+# Test for wetting-drying scheme functionality, using Thacker test case
+# Test case details in Gourgue et al (2009)
+
+from thetis import *
+import pytest
+import math
+
+@pytest.mark.parametrize("n,dt,alpha,max_err", [(25, 300., 4., 0.009), (10, 600., 8., 0.06)])

```

Comment:
You can add `ids`, e.g. `ids=['fine', 'coarse']`, to give human readable names to these configurations.